src/components/snapshot/overview/SnapshotExport.tsx

```diff
@@ -92,31 +94,25 @@ function SnapshotExport(props: SnapshotExportProps) {
     dispatch(resetSnapshotExport());
   };
 
-  const gsPathsCheckbox = !isProcessing ? (
-    <Checkbox checked={exportGsPaths} onChange={handleExportGsPathsChanged} />
-  ) : (
-    <Checkbox checked={exportGsPaths} disabled />
-  );
-
-  const canExport = userRoles.includes(SnapshotRoles.STEWARD);
-  const tooltipMessage =
```

Comment:
I did get rid of the tooltip, since it was specific to permission syncing.  This info is covered in helper text for the permission syncing checkbox.

(If we leave the tooltip out, we can also close this: https://broadworkbench.atlassian.net/browse/DR-2876)